pwndbg/commands/tls.py

```diff
@@ -84,6 +87,8 @@ def threads(num_threads, respect_config) -> None:
     headers = ["global_num", "name", "status", "pc", "symbol"]
     bold_green = lambda text: pwndbg.color.bold(pwndbg.color.green(text))
 
+    import gdb
```

Comment:
Is this needed? Is this ok? shouldn't this be protected with some if gdblib is available?